docker/entrypoint.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+# usage: file_env VAR [DEFAULT]
+#    ie: file_env 'XYZ_DB_PASSWORD' 'example'
+# (will allow for "$XYZ_DB_PASSWORD_FILE" to fill in the value of
+#  "$XYZ_DB_PASSWORD" from a file, especially for Docker's secrets feature)
+file_env() {
+    local var="$1"
+    local fileVar="${var}_FILE"
+    local def="${2:-}"
+    if [ "${!var:-}" ] && [ "${!fileVar:-}" ]; then
+        echo >&2 "error: both $var and $fileVar are set (but are exclusive)"
+        exit 1
+    fi
+    local val="$def"
+    if [ "${!var:-}" ]; then
+        val="${!var}"
+    elif [ "${!fileVar:-}" ]; then
+        val="$(< "${!fileVar}")"
+    fi
+    export "$var"="$val"
+    unset "$fileVar"
+}
+
+file_env 'IOTA_AUTH_USER'
+file_env 'IOTA_AUTH_PASSWORD'
+
+# Reading username and password at runtime
+read -p "Username : " IOTA_AUTH_USER
+read -p "Password : " IOTA_AUTH_PASSWORD
+echo "***********************************************"
+echo "WARNING: It is recommended to set IOTA Auth credentials(using difficult pattern) keeping security in mind and not to use default values"
+echo "These keys should be set using Docker Secrets"
+echo "***********************************************"
+
+#Setting values to ENV vars IOTA_AUTH_PASSWORD & IOTA_AUTH_USER
+export IOTA_AUTH_PASSWORD=$IOTA_AUTH_PASSWORD
+export IOTA_AUTH_USER=$IOTA_AUTH_USER
+
+npm install -g pm2
```

Comment:
Right line, wrong place. This already exists in the `Dockerfile` see https://github.com/telefonicaid/iotagent-json/blob/346886/docker/Dockerfile#L69

Please delete (as git will have been removed at this point in the operation

Reply:
Fixed with https://github.com/telefonicaid/iotagent-json/pull/380/commits/151094fb44fab57ec2ba01e6b9c170c31ceb3c42